tools/impeller.gni

```diff
@@ -10,7 +10,10 @@ declare_args() {
   impeller_enable_playground = false
 
   # Whether Impeller is supported on the platform.
-  impeller_supports_platform = is_mac || is_ios
+  impeller_supports_platform = is_mac || is_ios || is_linux
```

Comment:
@jonahwilliams is testing whether we can just add `is_windows` to this.

Reply:
`is_win` actually

Reply:
is_win

Reply:
I just changed this to `= true`.